### PR TITLE
Fix up design for recovery key creation dialog

### PR DESCRIPTION
**Pull request by dbkr**
Screenshot 2020-06-04 at 12 30 01
Screenshot 2020-06-04 at 12 30 20

Fixes vector-im/riot-web#13892
Fixes vector-im/riot-web#13893

Based on https://github.com/matrix-org/matrix-react-sdk/pull/4686

### CODE REVIEW

**Comment by t3chguy** [comment]
screenshots please to aid design review.

File: src/components/views/dialogs/BaseDialog.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*
 Copyright 2017 Vector Creations Ltd
 Copyright 2018, 2019 New Vector Ltd
-Copyright 2019 The Matrix.org Foundation C.I.C.
+Copyright 201, 2020 The Matrix.org Foundation C.I.C.
```
Comment (t3chguy) [inline src/components/views/dialogs/BaseDialog.js:4]: wat

Comment (dbkr) [reply src/components/views/dialogs/BaseDialog.js]: hah, oops

**Comment by aaronraimist** [comment]
What does "this login" mean? This session? It sounds more like something that helps you if you forget your password or something.

**Pull request closed without merging**